.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy Docs to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/gh-pages.yml'
+      - 'website/**'
+
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
I took this action code from https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus and I didn't pay super close attention to it.  It looks like setting the group limits the jobs to one at a time within the group.  Since the group name includes the `github.workflow`, it seems like it doesn't actually do anything.

Looking at action history it seems like both the CI and pages workflows were kicked off at the same time on my last commit.

Reply:
I was going to suggest we only run this action if the ci.yaml action succeeds but it seems like a lot of work... maybe later.